rmf_robot_sim_common/src/slotcar_common.cpp

```diff
@@ -621,33 +629,33 @@ std::pair<double, double> SlotcarCommon::update(const Eigen::Isometry3d& pose,
       const double d_yaw_tolerance = 5.0 * M_PI / 180.0;
       auto goal_heading = compute_heading(trajectory.at(_traj_wp_idx));
       double dir = 1.0;
-      displacements.second =
+      result.w =
         compute_change_in_rotation(current_heading, dpos, &goal_heading, &dir);
       if (dir < 0.0)
         current_heading *= -1.0;
 
       // If d_yaw is less than a certain tolerance (i.e. we don't need to spin
       // too much), then we'll include the forward velocity. Otherwise, we will
       // only spin in place until we are oriented in the desired direction.
-      displacements.first = std::abs(displacements.second) <
+      result.v = std::abs(result.w) <
         d_yaw_tolerance ? dir * dpos_mag : 0.0;
     }
   }
   else
   {
     const auto goal_heading = compute_heading(trajectory.back());
-    displacements.second = compute_change_in_rotation(
+    result.w = compute_change_in_rotation(
       current_heading,
       goal_heading);
 
     // Put in a deadzone if yaw is small enough. This essentially locks the
     // tires. COMMENTED OUT as it breaks rotations for some reason...
-    // if(std::abs(displacements.second) < std::max(0.1*M_PI/180.00, goal_yaw_tolerance))
+    // if(std::abs(result.w) < std::max(0.1*M_PI/180.00, goal_yaw_tolerance))
```

Comment:
Should there be a follow up to this? If so, we should add a TODO comment, otherwise, remove commented out code.

Reply:
Not sure it's been there since forever, since it seems not to cause any issue will just delete it.